app/views/wpcc/your_contributions/new.html.erb

```diff
@@ -4,7 +4,7 @@
   <h2 class="wpcc__heading">2. <%= t('wpcc.contributions.title') %></h2>
   <div class="contributions__row">
     <p class="t-contributions_description">
-      <%= t('wpcc.contributions.description', eligible_salary: @your_contribution.eligible_salary) %>
+      <%= t('wpcc.contributions.description', eligible_salary: @presenter.formatted_currency(@your_contribution.eligible_salary)) %>
```

Comment:
Do you think it would be better to wrap the contribution in a contribution presenter which handles the formatting? It would simplify how you refer to it in the view.